### PR TITLE
Merge feat/rosetta into master

### PR DESCRIPTION
**Pull request by andreibancioiu**
- Merge `feat/rosetta` into `master`

### CODE REVIEW

**Review by iulianpascalau — CHANGES_REQUESTED** [review]
We will merge **after** the merge in elrond-go

**Review by iulianpascalau — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Pull request merged**
